### DISCUSSION TITLE
web vault dark mode

### DISCUSSION THREAD

**Discussion by toxic0berliner** [dani-garcia/vaultwarden#2105] [Q&A] [answered]
I think the docker image is a bit behind in terms of web vault UI. Maybe it's not just as simple as pulling from upstream, but I find I'm missing the dark mode in options. 
I'm using the docker hub vaultwarden/server: latest
Server Installed Ok
    1.23.0 
Server Latest
    1.23.0 
Web Installed
    2.23.0c 

Thanks in advance for any time spent on this comfort point ;)

**Comment by BlackDex** [comment, marked as answer]
This is already fixed via https://github.com/dani-garcia/vaultwarden/commit/450c4d4d97da91898abc6a32bb748d275d36b6ac

When you use the testing tagged images it will be available.

**Reply by toxic0berliner** [reply]
Thanks for your kind answer ! I couldn't find much info, but I expect if I switch to testing I'll be unable to switch back to latest probably due to structural database upgrades you might do, am I correct ?
I have several users in my instance now, so if you could point me to a way of export/import that is sure to work to switch from testing back to latest in case of issues I'd be very thankful ;)

**Reply by BlackDex** [reply]
There shouldn't be any issues with switching back to a previous image currently. But as always, it is save to create a backup before you execute these kind of actions.

I suggest to read the following two pages:
- [Backing up your vault](https://github.com/dani-garcia/vaultwarden/wiki/Backing-up-your-vault)
- [General (not docker)](https://github.com/dani-garcia/vaultwarden/wiki/General-%28not-docker%29)
